src/components/Disputes/DisputeActions.js

```diff
@@ -342,9 +349,11 @@ const ANJRewardsMessage = () => {
   )
 }
 
-const VoteInfo = ({ outcome }) => {
+const VoteInfo = ({ commitmentDate, outcome, revealDate }) => {
   const theme = useTheme()
 
+  const formattedDate = dayjs(revealDate || commitmentDate).format(DATE_FORMAT)
```

Comment:
we have [this function ](https://github.com/aragon/court-dashboard/blob/77179b3a7a730728e41ae95b6701e2dd995665f8/src/utils/date-utils.js#L7)